setup_python.sh

```diff
@@ -84,6 +84,7 @@ conda activate ddpy2
 pip install -i https://pypi.python.org/simple pip==${DD_PIP_VERSION}
 pip install setuptools==${DD_SETUPTOOLS_VERSION}
 pip install -r requirements-py2.txt
+python3 -m pip install -r requirements.txt
```

Comment:
I would rather move this to line 68, just after the case statement, since it is not related to Python 2 in any way